x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -178,6 +181,9 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
               ...savedObjectTypes,
             ],
           },
+          alerting: {
+            all: [SIGNALS_ID, NOTIFICATIONS_ID],
+          },
```

Comment:
A note for SIEM - As you granted `all` privileges before I kept it as `all` here, but if you'd like, you can change this to `read` once this is merged, so  users with `read` to siem will also have `read` to alerting under siem.

Reply:
noted: https://github.com/elastic/kibana/issues/69698 thanks!